packages/design-system-react-native/package.json

```diff
@@ -72,11 +69,14 @@
     "@types/react": "^18.2.0",
     "@types/react-test-renderer": "^18",
     "babel-jest": "^29.7.0",
+    "bitcoin-address-validation": "2.0.0",
```

Comment:
We are pinning `bitcoin-address-validation` to major version `^2.0.0` to maintain compatibility with MetaMask Mobile, which uses `2.0.0` (see [metamask-mobile/package.json#L295](https://github.com/MetaMask/metamask-mobile/blob/main/package.json#L295)).